depthai_sdk/src/depthai_sdk/camera_control.py

```diff
@@ -0,0 +1,378 @@
+import logging
```

Comment:
Hi @Hutch07 ,
It seems like another file (camera_control.py) was created by mistake. Could you also target `develop` branch instead of `main`?